Makefile

```diff
@@ -15,8 +15,28 @@ SRCFILES := *.go
 all: firectl
 
 firectl: $(SRCFILES)
-	go build
+	go build -o firectl
 
+release: $(SRCFILES)
+	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build \
+		    -a \
+		    -installsuffix cgo \
+		    -ldflags "-s" \
+		    -gcflags=all=-trimpath=${TRIMPATH} \
```

Comment:
I don't see `TRIMPATH` defined anywhere in the Makefile.  Do you want the build to fail if it's not defined?  You can do that by adding `test -z $(TRIMPATH)` above the `go build` line.